src/main/java/ch/njol/skript/events/EvtCommand.java

```diff
@@ -37,7 +37,7 @@
 public class EvtCommand extends SkriptEvent { // TODO condition to check whether a given command exists, & a conditon to check whether it's a custom skript command
```

Comment:
The part of this todo comment mentionining checking if a command is a custom Skript command can be removed. You can also add something like CondIsCommandExists and remove this comment altogether.

Or just remove it without adding command exists because TODO comments are not used anymore, and instead feature requests are tracked from GitHub issues.

Reply:
I'm impressed you remembered that one, I can remove the TODO after merge

Reply:
Seems there was a PR (#3553) implementing `All Commands` with command information expressions which I didn't notice (Thanks to Frank for telling me about that)

I will keep this PR only for the `CondIsSkriptCommand` and try to see if there is a good way to check if command actually exists, because as I saw some plugins use `PlayerCommandPreprocessEvent` to make their commands work but it does not register them afaik, so this would not be accurate but I think it is their own problem not to register it.